.github/workflows/pr-is-up-to-date.yml

```diff
@@ -1,9 +1,13 @@
 name: PR is up-to-date
 on:
   pull_request_target:
-    branches: [ master ]
+    branches:
+      - master
+      - trunk
   push:
-    branches: [ master ]
+    branches:
+      - master
+      - trunk
```

Comment:
These changes are not sufficient. Lines 24, 38, 51, 60, 70, and 76 also need updating if this is to work right. And also line 11 in .github/files/pr-update-to.sh.

Reply:
I've just edited the PR description since I wasn't particularly clear about this - these changes aren't the only changes to be made in these files but are changes that can be made now. The additional changes would be in another PR, that can be merged at the time of the rename.

Reply:
Or if it's safer (if I'm misunderstanding how these are working) - all of the renames could happen at once at the time of the main rename. Happy to add this all to a separate PR if so.

Reply:
I think I may have gotten confused and this one isn't as bad as I thought either. On a closer look,

* A push to trunk would be processed as a tag push. Which means the pr-update-to tags wouldn't get updated and other PRs wouldn't get checked. OTOH, I'm guessing the "fix everything else to refer to trunk" PR is intended to be the first push (maybe we should use the release lock to ensure that?), and that would work fine.
* A PR based on trunk rather than master will get checked. But might be missing the checks against the per-project pr-update-to tags.

Reply:
Yes I realize I never considered that someone may accidentally push to trunk, and what repercussions that could have, so that's good to know.

> I'm guessing the "fix everything else to refer to trunk" PR is intended to be the first push (maybe we should use the release lock to ensure that?), and that would work fine.

Yeah that's right. On the Asana board for the project, under 'to do' I have a card for tasks that need to happen on the day. I'd like to have that more visible - possibly in a PR - or possibly editing the main project thread to add it. Currently the suggested flow is: release lock, wait for any actions to finish running, then rename the main branch in all mirror repos using the script, then rename the main branch for the Jetpack monorepo via the UI, followed afterwards by merging the PR with important/needed rename changes. Then merge the PR with all other relevant master to trunk wording changes (assuming that's ready to go, it may as well be merged here too). Then disable the release lock and let people know the change has happened.

Reply:
> Yes I realize I never considered that someone may accidentally push to trunk

We have direct pushing to master disabled in the monorepo's settings, GitHub will reject the attempt. GitHub should automatically migrate that to trunk when the rename happens. But every PR merge comes into Actions as a push to the merged-to branch.

> Currently the suggested flow is [...]

:+1: